modules/installation-aws-config-yaml.adoc

```diff
@@ -75,10 +81,14 @@ ifdef::gov[]
       - us-gov-west-1a
       - us-gov-west-1b
 endif::gov[]
-ifndef::gov,china[]
+ifdef::secret[]
+      - us-iso-east-1
```

Comment:
I have been unable to verify if the C2 secret regions have availability zones. I would have to think that the answer is yes, but can someone confirm it they follow the same syntax as public and gov regions? For example, would they be `us-iso-east-1a` and us-iso-east-1b`?

Reply:
Yes, the availability zones in the C2S partition follow the same naming pattern. For example, there is a `us-iso-east-1a` availability zone. I do not know how many availability zones there actually are, though.

Reply:
3 zones are listed, `us-iso-east-1a` and `us-iso-east-1b` would be good.
```
aws --region us-iso-east-1 ec2 describe-availability-zones
{
    "AvailabilityZones": [
        {
            "State": "available",
            "Messages": [],
            "RegionName": "us-iso-east-1",
            "ZoneName": "us-iso-east-1a"
        },
        {
            "State": "available",
            "Messages": [],
            "RegionName": "us-iso-east-1",
            "ZoneName": "us-iso-east-1b"
        },
        {
            "State": "available",
            "Messages": [],
            "RegionName": "us-iso-east-1",
            "ZoneName": "us-iso-east-1c"
        }
    ]
}
```

Reply:
@yunjiang29 `compute.platform.aws` and `controlPlane.platform.aws` each list the `us-iso-east-1a` and `us-iso-east-1b` availability zones. Thank you for confirming.